plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/OpenShiftUIActivator.java

```diff
@@ -106,4 +108,11 @@ public static void log(int status, String message, Throwable e) {
 	private String getId() {
 		return getBundle().getSymbolicName();
 	}
+
+    @Override
+    protected void initializeImageRegistry(ImageRegistry reg) {
+        super.initializeImageRegistry(reg);
+        reg.put(OpenshiftUIConstants.CHECKED_IMAGE_KEY, ImageDescriptor.createFromURL(this.getClass().getResource("/icons/checked.png")));
+        reg.put(OpenshiftUIConstants.UNCHECKED_IMAGE_KEY, ImageDescriptor.createFromURL(this.getClass().getResource("/icons/unchecked.png")));
+    }
```

Comment:
we have OpenShiftImages. Shouldnt we have an uniformous approach (where all images are images are in this class) for images?